docs-src/concepts/what-is-asynchronous-activity-completion.md

```diff
@@ -19,11 +19,19 @@ The Temporal Client can then be used to both Heartbeat Activity Execution progre
 
 #### When to use Async Completion
 
-The intended use-case for this feature is when an external system has the final result of a computation, started by an Activity.
+The intended use case for this feature is when an external system has the final result of a computation, started by an Activity.
+However, often it works better for the Activity to notify the external system, complete normally, and then wait in the Workflow for a [Signal](/concepts/what-is-a-signal) from the external system.
 
-Consider using Asynchronous Activities instead of Signals if the external process is unreliable and might fail to send critical status updates through a Signal.
+Consider using Async Completion when
 
-Consider using [Signals](/concepts/what-is-a-signal) as an alternative to Asynchronous Activities to return data back to a Workflow Execution if there is a human in the process loop.
-The reason is that a human in the loop means multiple steps in the process.
-The first is the Activity Function that stores state in an external system and at least one other step where a human would “complete” the activity.
-If the first step fails, you want to detect that quickly and retry instead of waiting for the entire process, which could be significantly longer when humans are involved.
+- the external process is unreliable and might fail to send critical status updates through a Signal
+- you want the external process to Heartbeat or receive Cancellation
+
+Consider using Signals when you want to immediatly retry a failure to notify the external system.
```

Comment:
I think this is not worded the best (we encourage retry policy to govern retries and you can set to immediate retry). I believe what @mfateev is saying in the internal thread is "Consider using Signals when you only want the activity success/fail/retry/timeouts to govern the _start_ of the operation rather than the whole operation". I'm not sure how best to word that.

Reply:
> we encourage retry policy to govern retries and you can set to immediate retry

this reads to you as not using a retry policy? perhaps:

Consider using Signals when you want to retry sooner a failure to notify the external system.

I think it's helpful to talk about the desired result here, and then get into details below

Reply:
Unsure how to word it best. I was just confused by the original statement which at first glance felt like it was saying "if you want to immediately retry failure, don't use async activity" which I don't think is what we wanted to say.